BLSQ/iaso
# Ia-1668: completeness table backend test suite

This PR provides a test suite for the completeness stats API so we can refactor / improve it with confidence.

No changes in behavior expected.

## Self proof reading checklist


## How to test

Not much to test manually. Run the test suite, and maybe check the  completeness stats page still works.
